### ISSUE TITLE
If Windows console support for UIA is enabled, console output cannot be reviewed using the review cursor and the navigator is locked in a list item with the class name "MultiaskingViewFrame".

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#15528]
## Notes
This is a replacement for #14377 and makes the description clearer.

Under advanced settings, Windows console support is set to "UIA". This bug cannot be reproduced if set to "Legacy".
### Steps to reproduce:
[testconsole.py.txt](https://github.com/nvaccess/nvda/files/12715627/testconsole.py.txt)

1. Use python to execute the above script in cmd
2. Alt+Tab to switch to other windows, and then back to the cmd window
3. Try to review the output in CMD
4. Press alt+Space, X to maximize the CMD window.
5. Alt+Tab to switch to other windows, and then back to the cmd window
### Actual behavior:
Perform step 3: You cannot use the review command to review the cmd window. At this time, the navigator’s window class name is 'MultiaskingViewFrame'.
Perform step 4: You will be able to review the CMD window again
Perform step 5: You cannot use the review command to review the cmd window. At this time, the navigator’s window class name is 'MultiaskingViewFrame'.
### Expected behavior:
The cmd output can be reviewed as expected.
### NVDA logs, crash dumps and other attachments:
[log.txt](https://github.com/nvaccess/nvda/files/12717419/log.txt)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA2023.3Beta3

#### Windows version:
Windows 10 22H2 (AMD64) build 19045.3448

#### Name and version of other software in use when reproducing the issue:
Windows CMD

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I can reproduce this using NVDA-2022.4
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by codeofdusk** [comment]
Windows 10 now allows the default terminal application to be changed. I'd suggest installing Windows Terminal and, in `wt` settings (<kbd>Control</kbd>+<kbd>,</kbd>), set your default terminal app to `wt`.

CC @carlos-zamora.

**Comment by cary-rowen** [comment]
@codeofdusk 
Hello, after my test, according to what you said, using Windows Terminal cannot solve this problem. Is there anything else I need to do?

**Comment by CyrilleB79** [comment]
@cary-rowen can you clarify if you make the tests with the options set to "Automatic (prefer UIA)" or "UIA when available"?
Unfortunately, this combobox is not a feature flag (was probably implemented before it), so things are less clear.

**Comment by cary-rowen** [comment]
@CyrilleB79 
Yes, I tested both options and the issue can be reproduced using Windows Terminal.

**Comment by seanbudd** [comment]
This issue is only caused by enabling an advanced preference in NVDA settings. It can be worked around by using a modern Windows Terminal. Closing as won't fix.

**Comment by codeofdusk** [comment]
Looking at this again, I think it's a duplicate of #12648. Unfortunately, I'm afraid UIA clients are just going to have to work around this one.

The "windows console support" option, as indicated in the user guide, does not affect terminal (which is always UIA).

**Comment by cary-rowen** [comment]
Hi @seanbudd 
As mentioned in my comment above, switching to a more modern terminal will not solve this problem.

**Comment by cary-rowen** [comment]
@codeofdusk 
Do you have any new discoveries about this?

**Comment by chripa85** [comment]
The problem persists on Windows 11. The 'legacy' fix under advanced settings alleviates the issue. Switching to Windows Terminal prevents the crash; however, it introduces new problems. The terminal becomes non-navigable, and text cannot be added when the mouse hovers over a line.

**Comment by SaschaCowley** [comment]
@chripa85 
> Switching to Windows Terminal prevents the crash; however, it introduces new problems. The terminal becomes non-navigable, and text cannot be added when the mouse hovers over a line.

What do you mean by this? I use Windows Terminal on a daily basis and have no problems navigating the terminal with NVDA. The review cursor works fine to review text, reading the command line works fine, and navigating the UI werks well as well.

**Comment by Adriani90** [comment]
@cary-rowen can you please test last NVDA alpha version on Windows 11 24 H2? Is this issue occuring still occuring?

**Comment by seanbudd** [comment]
Is this still an issue?

**Comment by cary-rowen** [comment]
I rarely encounter this now, and I'm in favor of close it for now, but I'll open it if I encounter it again.
